requirements.txt

```diff
@@ -1,3 +1,5 @@
+cherrypy
 codeclimate-test-reporter
 coverage
+nose
```

Comment:
Recommended testrunner for cherrypy app is `py.test`. I even had some tests stuck when using `nose`.